### ISSUE TITLE
Error with Windows 11 22h2 [0x80040155]

### ISSUE THREAD

**Issue by JeroenTuinstra** [phw198/OutlookGoogleCalendarSync#1543]
> Checklist:-
> * [x] [Upgrade](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Upgrading) to the latest release (alpha if available) ![latest release](https://img.shields.io/github/release/phw198/OutlookGoogleCalendarSync/all.svg)
> * [x] Reproduce the issue
> * [x] Complete the information below
> * [x] Provide your `OGcalsync.log` file (see [how-to](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file))

**OGCS Version**: unknown cannot start at all.   
**Edition**: Installed  
**Sync Direction**: O->G 

**Bug Description**  
When programme starts error: 0x80040155 "A problem was encountered with your office install"

**Steps to Reproduce the Issue**  
1. Start programme
2. error occurs
3. 
**Any other information**  
Any additional information that may help troubleshoot the issue.  
:paperclip: Attach your `OGcalsync.log` file.
Cannot start the programme, unsure where to find the log. 

Checked the wiki, but none of these entries exist: 
```
    HKEY_CLASSES_ROOT\Interface\{00063001-0000-0000-C000-000000000046}\TypeLib
    HKEY_LOCAL_MACHINE\SOFTWARE\Classes\WOW6432Node\Interface\{00063001-0000-0000-C000-000000000046}\TypeLib
    HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Classes\Interface\{00063001-0000-0000-C000-000000000046}\TypeLib
```

So error cannot be resolved with information provided. 
[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/9727282/OGcalsync.log)

**Comment by ArJay60** [comment]
Same problem over here.

**Comment by ArJay60** [comment]
Repairing Outlook did the job!

**Comment by phw198** [comment]
@ArJay60 Yep, that is the advice right at the [top of the wiki article](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/FAQs---COM-Errors#office-repair) for any COM based error :slightly_smiling_face: 

@JeroenTuinstra  If a repair doesn't work for you, there is still some value in following through steps 2 and 3 for `0x80040155`?
